src/uk/gov/hmcts/cmc/integrationtests/IntegrationTests.groovy

```diff
@@ -140,6 +146,19 @@ class IntegrationTests implements Serializable {
     env.SAUCELABS_USERNAME = 'civilmoneyclaimsT1'
     env.SAUCELABS_TUNNEL_IDENTIFIER = 'saucelabs-overnight-tunnel-cmc-T1'
 
+    String branchName = env.CHANGE_BRANCH ? env.CHANGE_BRANCH : env.BRANCH_NAME
+
+    if (branchName != 'master') {
+      for (repository in ['cmc/integration-tests', 'cmc/citizen-frontend', 'cmc/legal-frontend', 'cmc/claim-store-api']) {
+        String version = artifactoryClient.getLatestImageVersion(repository, branchName)
+
+        if (version != null) {
+          steps.echo "E2E: Docker image ${repository}:${version} has been found for branch ${branchName}"
```

Comment:
E2E? It is in a class called IntegrationTests

Reply:
Decided to use `E2E` term since this is what we should use more for that kind of tests do avoid confusion as the other tests in claim-store that uses database (API tests) are also called integration tests.

I agree this does not match well with other names in the repository. I am happy to change it to `IntegrationTests runner` or something.

Reply:
I would just remove the prefix

Reply:
Removed